bookwyrm/activitypub/response.py

```diff
@@ -0,0 +1,17 @@
+from django.http import JsonResponse
+
+from .base_activity import ActivityEncoder
+
+class ActivitypubResponse(JsonResponse):
+    """
+    A class to be used in any place that's serializing responses for Activitypub enabled clients.
+    Uses JsonResponse under the hood, but already configures some stuff beforehand
+    Made to be a drop-in replacement of JsonResponse
```

Comment:
sorry, this is such a nitpicky change, but I have a tiny laptop screen - can you line wrap at 80 characters?

Reply:
Of course :)
Done!